app/assets/stylesheets/components/common/_lists.scss

```diff
@@ -95,6 +95,15 @@
   @extend .list;
 }
 
+// Dough sets th, td to 14px. We want contained lists to match within editorial content.
+.editorial {
+  th, td {
+    ul, ol {
+      font-size: 14px;
```

Comment:
I'm just matching the line in basic.scss, so not sure I want to add anything extra.